extensions/standard-processors/processors/ListenUDP.cpp

```diff
@@ -0,0 +1,83 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "ListenUDP.h"
+
+#include "core/Resource.h"
+#include "core/PropertyBuilder.h"
+#include "controllers/SSLContextService.h"
+#include "utils/ProcessorConfigUtils.h"
+
+namespace org::apache::nifi::minifi::processors {
+
+const core::Property ListenUDP::Port(
+    core::PropertyBuilder::createProperty("Listening Port")
+        ->withDescription("The port to listen on for communication.")
+        ->withType(core::StandardValidators::get().LISTEN_PORT_VALIDATOR)
+        ->isRequired(true)
+        ->build());
+
+const core::Property ListenUDP::MaxQueueSize(
+    core::PropertyBuilder::createProperty("Max Size of Message Queue")
+        ->withDescription("Maximum number of messages allowed to be buffered before processing them when the processor is triggered. "
+                          "If the buffer is full, the message is ignored. If set to zero the buffer is unlimited.")
+        ->withDefaultValue<uint64_t>(10000)
+        ->isRequired(true)
```

Comment:
If it has a default value, does it make sense to require it? If it is empty, it will fall back to default.

Reply:
I think in C2 perspective it makes sense

Reply:
I think it helps in understanding the property, and NiFi does the same thing https://nifi.apache.org/docs/nifi-docs/components/org.apache.nifi/nifi-standard-nar/1.17.0/org.apache.nifi.processors.standard.ListenUDP/index.html (bold (required and has default value)